lib/BankAccount.php

```diff
@@ -71,9 +71,11 @@ public function instanceUrl()
      */
     public static function retrieve($_id, $_opts = null)
     {
-        $msg = "Bank accounts cannot be accessed without a customer ID or account ID. " .
-               "Retrieve a bank account using \$customer->sources->retrieve('bank_account_id') or " .
-               "\$account->external_accounts->retrieve('bank_account_id') instead.";
+        $msg = "Bank accounts cannot be retrieved without a customer ID or " .
+               "an account ID. Retrieve a bank account using " .
+               "`Customer::retrieveSource('customer_id', " .
+               "'bank_account_id')` or `Account::retrieveExternalAccount(" .
+               "'account_id', 'bank_account_id')`.";
```

Comment:
I know with have rules for a row max width but this makes it so hard to read :(

Reply:
Ah, sorry. We don't even enforce a max line length in stripe-php, I just try to keep everything under 80 columns out of habit.